.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Define owners for all files in the repository
+
+-       @m-goggins @bamader @ericbuckley
```

Comment:
Should we add Brady as well?

Reply:
Yep! Just waiting on his github username